### ISSUE TITLE
Accessibility update

### ISSUE THREAD

**Issue by EliRasske** [Team1091/team1091.github.io#6]
We need to add accessibility features & attributes; the following list is a few _starter ideas_ (in order of decreasing importance)

- Image `alt` text
- `aria-label`, `aria-role`, etc.
- Add high-contrast theme? (media query `prefers-contrast: more` )
- Print layouts (media query `@media print` ) for things like the calendar/sponsors page

Reply to this issue with any other accessibility changes that should be made.
